app/views/proposals/query.html.haml

```diff
@@ -0,0 +1,6 @@
+.inset
+  %h1
+    Proposals participated in
+    = datespan_header
+
+  = client_partial (current_user.client_slug || 'ncr'), "proposal_list", locals: {proposals: @proposals}
```

Comment:
That feels like a step in the wrong direction (towards more indirection). How about defaulting users to having that slug upon creation and migrating existing users? That'd make it painfully obvious what needs to change.